sound/soc/sof/ipc3-loader.c

```diff
@@ -0,0 +1,416 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
```

Comment:
commit message: should there be a note that all this ipc3-loader code is not new but extracted from loader.c? and that the duplication of code is handled in the following patches?

Reply:
@plbossart, you are right, I will update the commit message right away.